lms/templates/courseware/course_about.html

```diff
@@ -330,4 +337,41 @@ <h2>
 
 </%block>
 
+<%block name="course_about_details">
+<div class="container">
```

Comment:
This accurately follows the same mis-adjusted nesting as the edx-platform PR, so the code is correct, but difficult to read accurately.  Don't forget to remove the div.container from here if you remove it from the block in the edx-platform PR.

Reply:
Fixed (indented, and removed div.container)